Update python Azure Batch AI example doc

benalexkeen opened the pull request:
Corrected some typos. 

Also, I think the azure_file_share_name needs to be included in the path when mounting.

azurecla commented:
Hi __@stuartleeks__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

benalexkeen committed:
Update python Azure Batch AI example doc

Corrected some typos. 

Also, I think the azure_file_share_name needs to be included in the path when mounting.
